src/Tribe/Views/V2/Assets.php

```diff
@@ -634,4 +650,23 @@ public function should_enqueue_single_event_block_editor_styles() {
 
 		return true;
 	}
+
+	/**
+	 * Load assets on the add or edit pages of the block editor only.
+	 *
+	 * @since  TBD
+	 *
+	 * @return bool
+	 */
+	public function should_enqueue_admin() {
+		if ( ! is_admin() ) {
+			return false;
+		}
+
+		if ( ! get_current_screen()->is_block_editor ) {
+			return false;
+		}
```

Comment:
Let's ensure that the assets only load on our post type pages.

```suggestion
		}
		
		if ( ! tribe( 'admin.helpers' )->is_post_type_screen() ) {
			return false;
		}
```